src/roswire/proxy/roslaunch/reader.py

```diff
@@ -323,6 +332,19 @@ def _load_node_tag(self,
         cfg = cfg.with_node(node)
         return ctx, cfg
 
+    def _get_executable_type(self, path: str) -> ExecutableType:
+
+        executable_type = None
```

Comment:
This _should_ fail type checking. `executable_type` will either be inferred as `Optional[ExecutableType]` or `Any`. We can avoid the variable altogether and just return directly instead (e.g., `return ExecutableType.PYTHON`).

Reply:
Changed it.